netbout-web/src/test/java/com/netbout/dynamo/DyAliasITCase.java

```diff
@@ -67,4 +67,41 @@ public void makesAlias() throws Exception {
         );
     }
 
+    /**
+     * DyAlias can reject invalid email.
+     * @throws Exception If there is some problem inside
+     */
+    @Test (expected = Alias.InvalidEmailException.class)
+    public void rejectsInvalidEmail() throws Exception {
+        final Aliases aliases =
+            new DyBase().user(new URN("urn:test:13")).aliases();
+        final String name = "max";
+        aliases.add(name);
+        final Alias alias = aliases.iterate().iterator().next();
+        alias.email("test");
+    }
+
+    /**
+     * DyAlias can accept valid email.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void acceptsValidEmail() throws Exception {
+        final Aliases aliases =
+            new DyBase().user(new URN("urn:test:14")).aliases();
+        final String name = "jack";
+        aliases.add(name);
+        final Alias alias = aliases.iterate().iterator().next();
+        final String[] emails = {
+            "test@domain.com", "test@domain.com!test@domain.com",
```

Comment:
@dmzaytsev I would add a test for dots in the first email part, and sub domains in the other